Equipo2BigData/src/test/java/TestControlador/TestControlador.java

```diff
@@ -0,0 +1,396 @@
+		doReturn(controladorPanelPedidosMock).when(spyControlador).makeControladorPanelPedidos(any(Modelo.class),
+				any(Vista.class), any(Controlador.class));
+		doReturn(controladorPanelRegistroMock).when(spyControlador).makeControladorPanelRegistro(any(Modelo.class),
+				any(Vista.class), any(Controlador.class));
+		doReturn(controladorPanelAprovisionamientoMock).when(spyControlador)
+				.makeControladorPanelAprovisionamiento(any(Modelo.class), any(Vista.class), any(Controlador.class));
+		doReturn(controladorPanelTicketsMock).when(spyControlador).makeControladorPanelTickets(any(Modelo.class),
+				any(Vista.class), any(Controlador.class));
+		doReturn(controladorPanelFacturasMock).when(spyControlador).makeControladorPanelFacturas(any(Modelo.class),
+				any(Vista.class), any(Controlador.class));
+
+		when(modeloMock.getUser()).thenReturn(userMock);
+
+		when(userMock.getTipoLocal()).thenReturn("cafeteria");
+
+		when(modeloMock.getConexion()).thenReturn(conexionMock);
+
+		when(consultasMock.leerNumTransBBDD()).thenReturn(66);
+
+		when(modeloMock.getListaProductos()).thenReturn(listaProductosMock);
+
+		when(modeloMock.getUser()).thenReturn(userMock);
+
+		spyControlador.navegarPanelPedidos();
+
+		verify(spyControlador).controladorPanelPedidosMostrarPanelPedidos();
+	}
+
+	@Test
+	public void navegarPanelPedidosBAR() {
+		spyControlador = spy(new Controlador(modeloMock, vistaMock));
+		doReturn(controladorInicioMock).when(spyControlador).makeControladorPanelInicio(any(Modelo.class),
+				any(Vista.class), any(Controlador.class));
+		doReturn(controladorPanelPedidosMock).when(spyControlador).makeControladorPanelPedidos(any(Modelo.class),
+				any(Vista.class), any(Controlador.class));
+		doReturn(controladorPanelRegistroMock).when(spyControlador).makeControladorPanelRegistro(any(Modelo.class),
+				any(Vista.class), any(Controlador.class));
+		doReturn(controladorPanelAprovisionamientoMock).when(spyControlador)
+				.makeControladorPanelAprovisionamiento(any(Modelo.class), any(Vista.class), any(Controlador.class));
+		doReturn(controladorPanelTicketsMock).when(spyControlador).makeControladorPanelTickets(any(Modelo.class),
+				any(Vista.class), any(Controlador.class));
+		doReturn(controladorPanelFacturasMock).when(spyControlador).makeControladorPanelFacturas(any(Modelo.class),
+				any(Vista.class), any(Controlador.class));
+		doReturn(controladorPanelComandasMock).when(spyControlador).makecontroladorPanelComandas(any(Modelo.class),
+				any(Vista.class), any(Controlador.class));
+
+		when(modeloMock.getUser()).thenReturn(userMock);
+
+		when(userMock.getTipoLocal()).thenReturn("bar");
+
+		when(modeloMock.getConexion()).thenReturn(conexionMock);
+
+		when(consultasMock.leerNumTransBBDD()).thenReturn(66);
+		when(modeloMock.getListaProductos()).thenReturn(listaProductosMock);
+
+		when(modeloMock.getUser()).thenReturn(userMock);
+
+		spyControlador.navegarPanelPedidos();
+
+		// NO PUEDO VERIFICAR QUE ENTRE YA QUE NO DEBE ENTRAR
```

Comment:
Un test siempre tiene que hacer un assert, sino no es un test

Reply:
Sacado de la documentación oficial, teneis un ejemplo
//verify that method was never called on a mock
 verify(mockOne, never()).add("two");

Reply:
Gracias por el ejemplo, cambiado y comprobado